### PR TITLE
Add word2vec dataset, model, utils

### PR DESCRIPTION
**Pull request by mrgsv**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -7,12 +7,15 @@ scipy>=1.4.1
...
 # needed for plotly:
 nbformat>=4.2.0
 
 
 # ML
-scikit-learn>=0.23.2
\ No newline at end of file
+scikit-learn>=0.23.2
+torch~=1.9.0
+sberpm~=1.2.0
```
Comment (SberProcessMining) [inline requirements.txt:20]: Reuqirements file should be tuned - you want scikit-learn and torch to be the latest version, but that seems unnecessary. Also sberpm should not be requirenent for sberpm.

File: sberpm/ml/vectorizer/word2vec_utils/word2vec_dataset.py [hunk 2]
```diff
@@ -0,0 +1,74 @@
+from typing import List
+
+from torch.utils.data import Dataset
+
+from .word2vec_utils import Word2VecVocabulary, Word2VecException
+
+
+class Word2VecDataset(Dataset):
+    """Inspired by github.com/goddoe"""
```
Comment (SberProcessMining) [inline sberpm/ml/vectorizer/word2vec_utils/word2vec_dataset.py:9]: Are there any possible license conflicts?

**Review by SberProcessMining — CHANGES_REQUESTED** [review]

File: sberpm/ml/vectorizer/word2vec_utils/word2vec_dataset.py (hunk 2 above)
Comment (mrgsv) [reply sberpm/ml/vectorizer/word2vec_utils/word2vec_dataset.py:9]: There is no LICENSE file in this repo.

File: sberpm/ml/vectorizer/word2vec_utils/model.py [hunk 3]
```diff
@@ -1,40 +1,60 @@
 import torch
-from torch import nn
-import torch.nn.functional as F
+import torch.nn as nn
 
 
-class SkipGramModel(nn.Module):
```
Comment (SberProcessMining) [inline sberpm/ml/vectorizer/word2vec_utils/model.py]: Why have you dismissed a skip-gram model?

**Review by SberProcessMining — CHANGES_REQUESTED** [review]

File: sberpm/ml/vectorizer/word2vec_utils/model.py (hunk 3 above)
Comment (mrgsv) [reply sberpm/ml/vectorizer/word2vec_utils/model.py]: I suppose this model is slower than CBOW.

File: requirements.txt [hunk 4]
```diff
@@ -16,6 +16,5 @@ nbformat>=4.2.0
 
 # ML
 scikit-learn>=0.23.2
-torch~=1.9.0
```
[the remark this answers is not in the document]
Comment (mrgsv) [reply requirements.txt]: done

**Review by SberProcessMining — APPROVED** [review]

**Pull request merged**